weather-ai/src/lib/prompting/index.ts

```diff
@@ -9,11 +9,11 @@ export default class BuiltinPrompting {
     constructor(private session: AITextSession) {}
 
     async streamingPrompt(prompt: string): Promise<ReadableStream<string>> {
```

Comment:
Then, you don't need the `async` keyword. Same below

Reply:
Updated. PTAL.